### PR TITLE
PAYARA-4028 Add simple test to ensure values set in PostConstruct are not lost

### PR DESCRIPTION
**Pull request by Pandrex247**
Title
Associated PR: https://github.com/payara/Payara/pull/4136

### CODE REVIEW

**Comment by arjantijms** [comment]
LGTM

File: samples/clustered-singleton/clustered-singleton-ejb/pom.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <artifactId>clustered-singleton-ejb</artifactId>
+    <packaging>war</packaging>
+
+    <name>Payara Samples - Clustered Singleton EJB</name>
+
+    <build>
+        <finalName>clustered_singleton</finalName>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
```
Comment (dmatej) [inline samples/clustered-singleton/clustered-singleton-ejb/pom.xml]: Shouldn't it be configured in parent of all tests in pluginManagement part? And here in plugins only lines to <artifactId> if war lifecycle does not guarantee compilation already (I always forget if Y/N).

File: samples/clustered-singleton/clustered-singleton-ejb/src/main/java/fish/payara/samples/clustered/singleton/ejb/HelloServlet.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/*
```
Comment (dmatej) [inline samples/clustered-singleton/clustered-singleton-ejb/src/main/java/fish/payara/samples/clustered/singleton/ejb/HelloServlet.java]: Why is it split to two poms? The parent seems redundant to me.

Comment (dmatej) [reply samples/clustered-singleton/clustered-singleton-ejb/src/main/java/fish/payara/samples/clustered/singleton/ejb/HelloServlet.java]: Oh, I commented bad file ... sorry :)

**Comment by MarkWareham** [comment]
Requested @MattGill98  review as his changes to pom config went in before this

File: samples/clustered-singleton/clustered-singleton-ejb/src/main/java/fish/payara/samples/clustered/singleton/ejb/HelloServlet.java (hunk 2 above)
Comment (Pandrex247) [reply samples/clustered-singleton/clustered-singleton-ejb/src/main/java/fish/payara/samples/clustered/singleton/ejb/HelloServlet.java]: I split it into two as we currently don't have many samples for the clustered annotation, so I pre-empted the addition of some more by adding a parent.

File: samples/clustered-singleton/clustered-singleton-ejb/pom.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    <artifactId>clustered-singleton-ejb</artifactId>
+    <packaging>war</packaging>
+
+    <name>Payara Samples - Clustered Singleton EJB</name>
+
+    <build>
+        <finalName>clustered_singleton</finalName>
+        <plugins>
+            <plugin>
```
Comment (MattGill98) [inline samples/clustered-singleton/clustered-singleton-ejb/pom.xml]: Check the new parent - I think this block is redundant now ;)

**Pull request merged**